### PR TITLE
Update carrier shenanigans information and remove inaccurate information

### PR DESCRIPTION
**Pull request by TommyTran732**

### CODE REVIEW

File: content/posts/android/Android Tips.md [hunk 1]
```diff
@@ -144,17 +144,19 @@ GrapheneOS has the LTE only mode exposed in settings. You can set this by going
...
 
-I have seen several common suggestions in the privacy community to mitigate this problem which do not actually work:
+- Turn on Airplane mode. This will turn off the baseband modem and your phone will no longer broadcast the IMEI/IMSI to the nearby towers. Note that your phone can still talk to the tower regardless of whether a SIM card inside - this is why `911` calls work. Simply removing the SIM cards is not enough.
 
-- **Removing the SIM Card**: The baseband modem will continue to contact the cell towers with its IMEI to prepare for emergency calls. In fact, this is how you are able to call `911` even when you do not have a SIM card inserted.
+- Disable the eSIM management app after you have disabled all of the eSIMs. With certain carriers, the eSIM management app will connect to the provisioning server to check for eSIM updates.
```
Comment (wj25czxj47bu6q) [inline content/posts/android/Android Tips.md]: Do you mean the _privileged eSIM management toggle_ in GrapheneOS?

- The GOS toggle is very different from disabling an app. Figure out which, if either, achieves the intended result.
- If disabling this app turns out to be impossible on stock OS, then there's no point saying anything other than what works in GOS.
- You need to say what exact Google app is the relevant app here.

File: content/posts/android/Android Tips.md [hunk 2]
```diff
@@ -144,17 +144,19 @@ GrapheneOS has the LTE only mode exposed in settings. You can set this by going
...
 
-- **Removing the SIM Card**: The baseband modem will continue to contact the cell towers with its IMEI to prepare for emergency calls. In fact, this is how you are able to call `911` even when you do not have a SIM card inserted.
+- Disable the eSIM management app after you have disabled all of the eSIMs. With certain carriers, the eSIM management app will connect to the provisioning server to check for eSIM updates. 
 
-- **Using PGPP as a carrier**: The service randomizes your IMSI by regularly reprovisioning your eSIM. However, the IMEI broadcasted by the baseband modem would remain unchanged, allowing the carriers to track you anyways.
+**Note**: I have seen recommendations to use PGPP as a carrier to randomize the IMSI by regularly reprovisioning the eSIM. This is unlikely to be beneficial, as the IMEI broadcasted by the baseband modem would remain unchanged, allowing the carriers to track you anyways.
```
Comment (wj25czxj47bu6q) [inline content/posts/android/Android Tips.md:159]:
```suggestion
On a related note, I have seen recommendations to use PGPP as a carrier to randomize the IMSI by regularly reprovisioning the eSIM. This is unlikely to be beneficial, as the IMEI baked into the modem would remain unchanged, allowing carriers to track you anyways.
```

File: content/posts/android/Android Tips.md [hunk 3]
```diff
@@ -252,7 +254,8 @@ Google Fi provides [opportunistic end&#8209;to&#8209;end encryption](https://fi.
 
 This is not without its caveats:
 - Google Fi requires Play Services and the [Fi app](https://play.google.com/store/apps/details?id=com.google.android.apps.tycho&hl=en_US) to work properly. Without Play Services, all of the features mentioned above, along with visual voicemail will not work. SMS messages will have random strings added at the end of each of them.
-- On GrapheneOS, Fi VPN and end to end encrypted calls with Fi will not work. Fi VPN requires privileged integration with the operating system which GrapheneOS developers are not willing to bundle, so it will likely never work in the foreseeable future. I am not entirely sure why end to end encrypted calls with FI are not working at the moment.
+- On GrapheneOS, Fi VPN will not work. Fi VPN requires privileged integration with the operating system which GrapheneOS developers are not willing to bundle, so it will likely never work in the foreseeable future.
```
Comment (wj25czxj47bu6q) [inline content/posts/android/Android Tips.md:257]:
```suggestion
- On GrapheneOS, Fi&nbsp;VPN will not function. Fi&nbsp;VPN requires privileged integration with the operating system which is unlikely to be supported for the foreseeable future. 
```

File: content/posts/android/Android Tips.md [hunk 4]
```diff
@@ -252,7 +254,8 @@ Google Fi provides [opportunistic end&#8209;to&#8209;end encryption](https://fi.
 
 This is not without its caveats:
 - Google Fi requires Play Services and the [Fi app](https://play.google.com/store/apps/details?id=com.google.android.apps.tycho&hl=en_US) to work properly. Without Play Services, all of the features mentioned above, along with visual voicemail will not work. SMS messages will have random strings added at the end of each of them.
-- On GrapheneOS, Fi VPN and end to end encrypted calls with Fi will not work. Fi VPN requires privileged integration with the operating system which GrapheneOS developers are not willing to bundle, so it will likely never work in the foreseeable future. I am not entirely sure why end to end encrypted calls with FI are not working at the moment.
+- On GrapheneOS, Fi VPN will not work. Fi VPN requires privileged integration with the operating system which GrapheneOS developers are not willing to bundle, so it will likely never work in the foreseeable future. 
+- Google Fi Wifi-calling does not work behind a VPN.
```
Comment (wj25czxj47bu6q) [inline content/posts/android/Android Tips.md]: Needs to be more specific. Does not work when _any_ VPN profile is enabled? Does not work when the VPN profile advertises a route for all traffic? Does not work when the OS is configured to block traffic outside of the VPN?

**Review by wj25czxj47bu6q — CHANGES_REQUESTED** [review]

File: content/posts/android/Android Tips.md (hunk 4 above)
Comment (TommyTran732) [reply content/posts/android/Android Tips.md]: Okay, so
- The Fi app gotta be in the owner profile iirc, dont even mention it
- It only breaks with the killswitch on
- if you dont disable the killswitch, you don't even need split tunneling

File: content/posts/android/Android Tips.md (hunk 1 above)
Comment (TommyTran732) [reply content/posts/android/Android Tips.md]: It just means the esim management app. I am not sure how it works on random android devices. I guess we can just frame this for GOS for now, what u think?

Comment (wj25czxj47bu6q) [reply content/posts/android/Android Tips.md]: I still consider the package name/ID required information before merging this.

Comment (TommyTran732) [reply content/posts/android/Android Tips.md]: I don't, this is just gonna sit for eternity before I can pull the correct package ID, and its is different depending on the OS. Not everyone uses the Google one.

**Comment by wj25czxj47bu6q** [comment]
For future reference: The Wireshark display filter `dns || esp || udpencap || isakmp` makes it easy to see ePDG traffic and DNS queries.

**Pull request merged**